source/NVDAObjects/UIA/edge.py

```diff
@@ -491,6 +490,16 @@ def _get__isTextEmpty(self):
 			return True
 		return False
 
+	def _get_landmark(self):
+		ariaRoles=self._getUIACacheablePropertyValue(UIAHandler.UIA_AriaRolePropertyId).lower()
+		landmarkId=self._getUIACacheablePropertyValue(UIAHandler.UIA_LandmarkTypePropertyId)

```

Comment:
Do you actually use landmarkID? I'd think you'd want to fetch this right at the top, and return early if this is 0 before even needing to fetch ariaRole